plugins/timestampTrade/timestampTrade.py

```diff
@@ -0,0 +1,85 @@
+import stashapi.log as log
+from stashapi.stashapp import StashInterface
+import stashapi.marker_parse as mp
+import os
+import sys
+import requests
+import json
+
+per_page = 100
+
+def processScene(s):
+    if len(s['stash_ids']) > 0:
+        for sid in s['stash_ids']:
+            #                print('looking up markers for stash id: '+sid['stash_id'])
```

Comment:
From your plugin description it seems that you want the stash_ids from `stashdb`? Don't you need to check the endpoint to make sure you dont get stash_ids from another stash-box instance? Not sure if there is another public/private stash-box instance except stashdb (maybe tpdb) but it doesnt hurt making sure

Reply:
The database does keep track of stashid and endpoints on the api side but for simplification all endpoints will hit the same api.
I'm assuming as uuid are configured to  generate mostly randomly there will not be any collisions between two instances, while technically possible the amount of possible numbers it's like winning the lotto jackpot several times in a row.
The api does return the endpoint and stashid so the plugin can be updated to verify that the stashid does point to the right endpoint before accepting the results.